loader/loader.c

```diff
@@ -67,6 +67,9 @@
 #if defined(WIN32)
 #include "loader_windows.h"
 #endif
+#ifdef LOADER_ENABLE_LINUX_SORT
+#include "loader_linux.h"
+#endif  // LOADER_ENABLE_LINUX_SORT
```

Comment:
Is `loader_linux.h` going to be used for non-sorting linux only behavior? Seems odd to have it behind a macro named LOADER_ENABLE_LINUX_SORT.

Reply:
We only currently use it for sorting.  The #ifdef just makes it easier.  However, I noticed that we really don't need this #ifdef and the one above it for Win32 because the source files themselves are wrapped in the proper #ifdef.  So I'm removing both of these.

Reply:
Be careful about removing them, GN builds use the same headers and it was something I didn't know we had to take into account.

Reply:
I kept them, but added a comment on why I'm using that define around the loader_linux.h file for now.  Figured that's the safest change.